src/main/resources/io/jenkins/plugins/worktile/WTDeployNotifier/help-releaseName.html

```diff
@@ -1,8 +1,8 @@
 <div>
-    发布的名称。名称中可以使用环境变量进行动态变量替换，例如： release-${BUILD_ID}, 表示利用构建id动态生成发布名称。
+    The name of the release. You can use environment variables for dynamic variable substitution in the name.
```

Comment:
The name of the release. You can use Jenkins environment variables to format with the sytanx ${JENKIN_VAR}

For example: you can use release-${BUILD_ID} to use Jenkins-auto-generated build id as part of the release name with ${BUILD_ID}. If the environment variable does not exist, the source character will be used.